client/src/app/services/alert.service.ts

```diff
@@ -0,0 +1,30 @@
+import { Injectable } from '@angular/core';
+import { MatSnackBar } from '@angular/material/snack-bar';
+import { SnackbarComponent } from '../components/snackbar/snackbar.component';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class AlertService {
+  constructor(private _snackBar: MatSnackBar) {}
+
+  openSnackBar(style: string, message: string) {
+    this._snackBar.openFromComponent(SnackbarComponent, {
+      duration: 5000,
+      panelClass: [style],
+      data: { message: message },
+    });
+  }
+
+  openErrorMessage(message: string) {
```

Comment:
I guess we can rename methods to 'info', 'error', 'success'
or even delete them and use only one and pass type of message as parameter like show('info', message)